azurelinuxagent/common/osutil/default.py

```diff
@@ -266,27 +266,27 @@ def remove_legacy_firewall_rule(self, dst_ip):
 
     def enable_firewall(self, dst_ip, uid):
         """
-        It checks if every iptable rule exists and add them if not present. It returns a tuple(enable firewall success status, update rules flag)
+        It checks if every iptable rule exists and add them if not present. It returns a tuple(enable firewall success status, missing rules array)
         enable firewall success status: Returns True if every firewall rule exists otherwise False
-        update rules flag: Returns True if rules are updated otherwise False
+        missirg rules: array with names of the missing rules ("ACCEPT DNS", "ACCEPT", "DROP")
```

Comment:
Maybe we should clarify in a comment here that these are the missing rules before reapply operation.

Reply:
any suggestions? "It checks if every iptable rule exists" above seems enough to me

Reply:
something like?
```suggestion
        missing rules: array with names of the missing rules before attempt to add them ("ACCEPT DNS", "ACCEPT", "DROP")
```
I was just thinking of case where we attempt to re-add the rules and some get added but agent fails to add others. Then we should clarify if the missing rules array was created before or after that attempt.

It's clear from the code though, so might be overkill to add the comment.

Reply:
OK, I'll leave it as is. Anyways, all this code is already changed in the fix for nftables and this return value is no longer needed.